src/containers/flows/TransferOwnership/TransferOwnership.tsx

```diff
@@ -57,6 +58,7 @@ const TransferOwnership = ({
 	const isOwnersAddress = ownerId.toLowerCase() === walletAddress.toLowerCase();
 	const hasInputError = inputError !== '';
 	const valid = isValid(walletAddress);
+	const zna = (ROOT_DOMAIN.length ? ROOT_DOMAIN + '.' : '') + domainName;
```

Comment:
This was a weird one because the other flows that use the Wizard ```NFTDetails``` didn't require a hacky line to re-append the ROOT_DOMAIN. Looked into it and found ```TransferOwnership``` prop ```domainName``` was being passed ```domainRaw```. I've swapped this out [here](). All looks good, but I wonder if there was a reason the raw domain was being passed - doesn't seem to be, does that all look good to you ?

Reply:
Hmm not sure, the transfer flow was made so long ago. As long as the zna is correct in the UI we can approve it for now, as we'll be refactoring these flows when we migrate to zOS.